### PR TITLE
Enforce snake case and various minor fixes

### PR DESCRIPTION
**Pull request by kierio04**
Review should check if all imports have been accounted for. A good majority of code ported from LUA core is still in camel case, however that's beyond the scope of this PR (and would require a good deal more work).

### CODE REVIEW

File: scripts/Modules/TTK_Lib.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 import zlib
 
 from .framesequence import FrameSequence
-from . import TTK_config
+from . import ttk_config
```
Comment (malleoz) [inline scripts/Modules/TTK_Lib.py:10]: If you're going to change capitalization here, you should rename the file too.

Comment (malleoz) [reply scripts/Modules/TTK_Lib.py:10]: This might requite a `git mv`

File: scripts/RMC/save_ghost_to_both_csv.py [hunk 2]
```diff
@@ -1,8 +1,8 @@
 from dolphin import gui
-from Modules import TTK_Lib
+from Modules import ttk_lib
```
Comment (malleoz) [inline scripts/RMC/save_ghost_to_both_csv.py:2]: Same as earlier comment

File: scripts/Modules/mkw_utils.py [hunk 3]
```diff
@@ -48,8 +48,9 @@ def delta_position(playerIdx=0):
 
     return physics_ref.position() - dynamics_ref.position()
 
-"""Next 3 functions are used for exact finish display"""
-def getIGT(lap, player):
+# Next 3 functions are used for exact finish display
+
+def get_igt(lap, player):
```
Comment (vabold) [inline scripts/Modules/mkw_utils.py:53]: If we are going to enforce the snake case function rule, it should be done repository-wide in this PR.

File: scripts/RMC/Wheelie_Turn_Left.py
Comment (ximk) [inline scripts/RMC/Wheelie_Turn_Left.py:1]: Why is this file being left capitalized?

File: scripts/RMC/Wheelie_Turn_Right.py
Comment (ximk) [inline scripts/RMC/Wheelie_Turn_Right.py:1]: see comments from wheelie_turn_left

File: scripts/Modules/TTK_config.py
Comment (ximk) [inline scripts/Modules/TTK_config.py:1]: see @malleoz's comment on TTK_Lib

**Review by ximk — CHANGES_REQUESTED** [review]
Consistency issues.

**Comment by kierio04** [comment]
Addressed all changes requested, so should be good to merge now.

**Review by ximk — APPROVED** [review]

**Review by ximk — CHANGES_REQUESTED** [review]
Will reapprove when fixes are made by myself.

**Review by ximk — APPROVED** [review]
Scripts should work fine now.

**Review by malleoz — DISMISSED** [review]

**Review by malleoz — APPROVED** [review]
Tested and all scripts working on my end.

**Pull request merged**